### ISSUE TITLE
Keeps rich presence active upon tidal closing?

### ISSUE THREAD

**Issue by NocFA** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#3]
The application appears to continue to output a rich presence signal even after tidal has been closed and is no longer in use, I have to close it and reopen to fix my status.

**Comment by purpl3F0x** [comment]
It keeps displaying it forever or just for some minutes ? 
(Also I guess the remaining time goes to 0 and stays there ?)

**Comment by purpl3F0x** [comment]
I'm closing the issue, since got no response.

**Comment by NocFA** [comment]
Sorry for not responding, had a hectic few weeks at work.
The issue appears to stem from when tidal is closed and reopened while a song was playing or something along those lines, for instance right now I was listening to something, it was showing that just fine, and the song after that.
I then quit tidal, it said that the music had stopped. 

Upon reopening tidal the application no longer functioned at all, it still said that song was stopped and the only way for it to connect and be correct again is to reopen the application.

**Comment by purpl3F0x** [comment]
Yeah i think that might be an issue. I will take a look at it , since i now use roon and not tidal for playback.

**Comment by NocFA** [comment]
That'd be awesome, but sounds like my AV is gonna fucking hate it lmao,
Also, how's roon? I've given it a try but I don't think I really knew what to do with it at all if I'm honest, what makes it special?

**Comment by purpl3F0x** [comment]
OK I found the issue, it seems that tidal isn't really closing  anymore and it leaves some leftovers behind. And the app was getting that tidal is still opened.I fixed that. I will upload the binary in a minute or so.
Let me know

Off Topic:
Roon is very handy, especially if you have local library (or if you also use Qobuz, which I don't). It's not that its a complex program but it has a lot of features(network/ mutliroom streaming/  live lyrics etc...). 
It's more like a thing it's not necessary , but if you start using you can't do without out.... :P

**Comment by purpl3F0x** [comment]
https://github.com/purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL/releases/tag/v.1.1
